app/src/sharedTest/java/org/oppia/android/app/splash/SplashActivityTest.kt

```diff
@@ -381,6 +381,17 @@ class SplashActivityTest {
     intended(hasComponent(OnboardingActivity::class.java.name))
   }
 
+  @Test
+  fun testSplashActivity_hasCorrectActivityLabel() {
+    initializeTestApplication()
+
+    activityTestRule.launchActivity(null)
+    testCoroutineDispatchers.advanceUntilIdle()
+
+    val title = activityTestRule.activity.title
+    assertThat(title).isEqualTo(context.getString(R.string.splash_activity_title))
```

Comment:
Now that you are using `app_name` as label please remove `splash_activity_title ` from strings.xml and use `app_name` in this line.

Also, please make sure that you run the test cases. I think you have written this test cases but did not ran it as this test case is failing currently.

Reply:
Okay sir, made that changes